packages/perseus-editor/src/widgets/python-program-editor.tsx

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable no-useless-escape */
+/**
+ * This editor is for embedding Khan Academy Python programs.
+ */
+
+import {
+    components,
+    Changeable,
+    EditorJsonify,
+} from "@khanacademy/perseus";
+import PropTypes from "prop-types";
+import * as React from "react";
+import _ from "underscore";
```

Comment:
Related to above, we're working to remove our dependance on `underscore`. Can we use modern, ES6 patterns instead?